ruby-stellar-base.gemspec

```diff
@@ -16,19 +16,19 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["generated", "lib"]
 
   spec.add_dependency "xdr", "~> 3.0.0"
-  spec.add_dependency "digest-crc"
-  spec.add_dependency "base32"
+  spec.add_dependency "digest-crc", ">= 0.5"
+  spec.add_dependency "base32", ">= 0.3"
   spec.add_dependency "rbnacl", ">= 6.0"
   spec.add_dependency "activesupport", ">= 5.0.0"
```

Comment:
What's the motivation for sticking with `>=` on these and not being somewhat pessimistic that a major version change might break our use of them? I'd favor `~>` here too unless we have tons of confidence about the future of them.

Reply:
good catch! Using the fish makes sense

Reply:
I was just sicking with the pattern we had for the rest of the non-development gems. Should I adjust all the dependencies to use `~>`?

Reply:
Looks like using the fish syntax makes the build fail. Multiple gems depend on different versions of another gem, and the ~> is more restrictive and doesn't allow a later version to be installed.